packages/ocaml-variants/ocaml-variants.5.0.0~rc1+tsan/opam

```diff
@@ -0,0 +1,76 @@
+opam-version: "2.0"
+license: "LGPL-2.1-or-later WITH OCaml-LGPL-linking-exception"
+synopsis: "First release candidate of OCaml 5.0.0, with ThreadSanitizer instrumentation"
+maintainer: "platform@lists.ocaml.org"
+authors: ["Xavier Leroy" "Damien Doligez" "Alain Frisch" "Jacques Garrigue" "Didier Rémy" "Jérôme Vouillon" "Anmol Sahoo" "Olivier Nicole" "Fabrice Buoro"]
+homepage: "https://github.com/ocaml-multicore/ocaml-tsan"
+bug-reports: "https://github.com/ocaml-multicore/ocaml-tsan/issues"
+dev-repo: "git+https://github.com/ocaml-multicore/ocaml-tsan.git#tsan_5.0.0-rc1"
+depends: [
+  "ocaml" {= "5.0.0" & post}
+  "base-unix" {post}
+  "base-bigarray" {post}
+  "base-threads" {post}
+  "base-domains" {post}
+  "base-nnp" {post}
+  "ocaml-beta" {opam-version < "2.1.0"}
+]
+conflict-class: "ocaml-core-compiler"
+flags: [ compiler avoid-version ]
+setenv: CAML_LD_LIBRARY_PATH = "%{lib}%/stublibs"
+build-env: [
+  [LSAN_OPTIONS = "detect_leaks=0,exitcode=0"]
+  [ASAN_OPTIONS = "detect_leaks=0,exitcode=0"]
+]
+build: [
+  [
+    "./configure"
+    "--prefix=%{prefix}%"
+    "--docdir=%{doc}%/ocaml"
+    "-C"
+    "--disable-flat-float-array" {ocaml-option-no-flat-float-array:installed}
+    "--enable-flambda" {ocaml-option-flambda:installed}
+    "--enable-frame-pointers" {ocaml-option-fp:installed}
+    "CC=cc" {!ocaml-option-32bit:installed & !ocaml-option-musl:installed & (os="openbsd"|os="macos")}
+    "CC=musl-gcc" {ocaml-option-musl:installed & os-distribution!="alpine"}
+    "CFLAGS=-Os" {ocaml-option-musl:installed}
+    "--enable-tsan" {ocaml-option-tsan:installed}
```

Comment:
This package doesn't exist - should the `{ocaml-option-tsan:installed}` part have been deleted?

Reply:
I might have misunderstood something; I thought that `ocaml-option-tsan` was a virtual package that would be considered installed if and only if there was a `+tsan` in the switch name.

But now that you mention it, yes, it probably makes sense to do `--enable-tsan` unconditionally.